app/components/containers/containers.html

```diff
@@ -8,13 +8,12 @@
   <rd-header-content>Containers</rd-header-content>
 </rd-header>
 
-<div class="col-lg-12">
+<div st-table="displayedContainers" st-safe-src="containers" class="col-lg-12">
   <rd-widget>
     <rd-widget-header icon="fa-server" title="Containers">
       <div class="pull-right">
         Items per page:
-        <select ng-model="state.pagination_count" ng-change="changePaginationCount()">
-          <option value="0">All</option>
+        <select rd-pagination="containers" ng-model="state.pagination_count">
           <option value="10">10</option>
```

Comment:
Pagination values are the same for each table, could this be integrated inside the pagination directive?